src/Map/src/Bridge/Google/src/GoogleOptions.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Bridge\Google;
+
+use Symfony\UX\Map\Bridge\Google\Option as GoogleOption;
+use Symfony\UX\Map\MapOptionsInterface;
+
+/**
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+final class GoogleOptions implements MapOptionsInterface
+{
+    public function __construct(
+        private ?string $mapId = null,
+        private GoogleOption\GestureHandling $gestureHandling = GoogleOption\GestureHandling::Auto,
```

Comment:
Do we need the namespace alias ? I'm not sure it brings much there, and afaik it's not in Symfony code style (?)

Reply:
They use namespace alias for Constraints, e.g.: https://symfony.com/doc/current/reference/constraints/NotBlank.html

Reply:
Not sure it's the same thing here.. :|